packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.5
```

Comment:
There is a non-breaking public api change, this should be 1.1.0

Reply:
fixed